src/OptionsRESTController.php

```diff
@@ -170,6 +217,17 @@ public function get_item_schema() {
 					Option::ALL_TEMPLATES_SUPPORTED => [
 						'type' => 'boolean',
 					],
+					self::SUPPRESSIBLE_PLUGINS      => [
+						'type'     => 'object',
+						'readonly' => true,
+					],
+					Option::SUPPRESSED_PLUGINS      => [
+						'type' => 'object',
+					],
+					self::DATE_FORMAT               => [
+						'type'     => 'string',
+						'readonly' => true,
+					],
```

Comment:
Isn't the date format already available via core's settings endpoint?

Reply:
Yes, good point. Update forthcoming.

Reply:
Added a `SiteSettingsProvider` component in cf61f3836

Works the same:
<img width="409" alt="Screen Shot 2020-07-14 at 9 05 03 PM" src="https://user-images.githubusercontent.com/9020968/87495018-05605180-c616-11ea-9ca7-810d44a252a6.png">